docs/src/pages/careers/react-engineer-x-charts.md

```diff
@@ -0,0 +1,125 @@
+
+### Skills you should have
+
+- **Expertise in the modern JavaScript ecosystem**.
+  MUI is built on the shoulders of giants, making use of technologies such as ES2021, TypeScript, Node.js, React, Next.js, webpack, and Babel.
+- **A track record of demonstrating an eye for product and solving real-world user problems**. If you have a knack for solving problems at the root cause, shipping beautiful user interfaces and intuitive APIs, we want you on our team.
+- **Experience building and shipping production code in a team setting** with a passion for writing tested, performant, and high-quality code.
+- **Strong written and verbal communication skills**.
+  As part of the team, you'll interface both directly and indirectly with community members and enterprise customers, and contribute to user documentation. Clear communication is fundamental in creating intuitive and compelling resources.
+- **Ability to dive into complex problems**.
+  You should be able to quickly assess, understand, and iterate upon aspects of our codebase.
+- **Ready and willing to ask and answer questions**.
+  If you're comfortable saying you're unsure, asking for help; but equally reaching out to assist others, you'll be an incredible addition to our team. We thrive because of continuous learning. First-time mistakes should be celebrated, not blamed.
+- **Avoid monolithic deliverables**.
+  You scope and stage your work into well-defined milestones to ship.
+- **Past work with front-end infrastructure**.
+  Perhaps you've created your company's design system, written a Babel plugin, or written complex React components.
+  It would be great if you could address this in your cover letter!
+
+### What it would be nice if you had, but isn't required
+
+- **You've contributed to a design system before**.
+  You have worked on enterprise components in the past, maybe for internal use in your company.
+  You have built complex and advanced features while making sure the accessibility was fully functional.
+- **You've maintained an active repository before**.
+  Maybe you've helped maintain a popular open-source repository, or perhaps you've worked on internal repositories that saw contributions from multiple teams.
+  Previous experience with highly active repository workflows is a definite plus for this role.
+- **You have contributed code to MUI before**. A history of contributing to MUI would be a definite plus.
+
+## Benefits & Compensation
+
+Competitive compensation depending on the profile and location.
+We are ready to pay top market rates for a person that can clearly exceed the role's expectations.
+You can find the other perks & benefits on the [careers](https://mui.com/careers/#perks-amp-benefits) page.
+
+## How to apply?
+
+[Apply now for this position 📮](https://jobs.ashbyhq.com/MUI/77584e4c-9379-45f3-a294-6e50bf250383/application?utm_source=ZNRrPGBkqO)
```

Comment:
Should we have a dedicated role for `React Engineer - xCharts` on Ashby or we can get away with the current `React Engineer - X` one? My main argument is that if we find someone `who` is a solid candidate but does not have a prior experience with charts we can rotate people around and still be ok with the hire. I'm open to suggestions.

Reply:
I think the job in Ashby should be a new "React Engineer - xCharts" to match with this page. We can transfer candidates between Ashby job.

> My main argument 

Which option are you arguing for 😁?

Reply:
My point was if we should have an opening specific for charts on the general MUI X React Engineer one but already discussed it and the charts it is. We can open the React Engineer - X general role just to differentiate the candidates.